README.md

```diff
@@ -401,17 +423,21 @@ def lik(data: HMMData, theta: HMMParam) -> float:
     k1, x, z = data
     k2, pi, T, E = theta
     assert k1 == k2
-    
-    # FIXME: compute the likelihood
-    p = pi[z[0]]
-    for i, s in enumerate(z[1:]):
-        p *= T[z[i], s]
-    for i, _ in enumerate(z):
-        p *= E[z[i], x[i]]
-    return p
 
+    for i in range(len(z)):
+        if i == 0: 
+            count = pi[z[i]]*E[z[i]][x[i]]
+        else: 
+            count = count * T[z[i-1]][z[i]]*E[z[i]][x[i]]
+
+    return count
```

Comment:
Maybe `count` isn't the most obvious name here, considering that it is a probability, but I'll allow it.